runtime/doc/treesitter.txt

```diff
@@ -1207,7 +1211,12 @@ LanguageTree:tree_for_range({range}, {opts})
         TSTree|nil
 
 LanguageTree:trees()                                    *LanguageTree:trees()*
-    Returns all trees this language tree contains. Does not include child
-    languages.
+    Returns all trees of the regions parsed by this parser. Does not include
+    child languages. The result is list-like if
```

Comment:
If it's not list-like, what is the caller supposed to expect?

Reply:
```
        table<integer, TSTree>
```